common/test/acceptance/pages/lms/programs.py

```diff
@@ -20,3 +20,11 @@ def are_cards_present(self):
     def is_sidebar_present(self):
         """Check whether sidebar is present."""
         return self.q(css='.sidebar').present
+
+
+class ProgramDetailsPage(PageObject):
+    """Program details page."""
+    url = BASE_URL + '/dashboard/programs/123'
```

Comment:
@schenedx the bok choy tests need a valid URL which they can use to hit the details page.